lms/urls.py

```diff
@@ -185,15 +185,15 @@
         # First we include views from course_wiki that we use to override the default views.
         # They come first in the urlpatterns so they get resolved first
         url('^wiki/create-root/$', course_wiki_views.root_create, name='root_create'),
-        url(r'^wiki/', include(wiki_pattern())),
-        url(r'^notify/', include(notify_pattern())),
+        url(r'^wiki/', wiki_pattern()),
```

Comment:
While this technically works, it isn't recommended usage and bypasses the validation code in `include()`.  A better fix might be something like `wiki_urlpatterns, wiki_app_name, wiki_namespace = wiki_pattern()` before the `urlpatterns` declaration, and then `url(r'^wiki/', include((wiki_urlpatterns, wiki_app_name), namespace=wiki_namespace)),` inside it.

`admin.site.urls` is a special case where the URL configuration is generated by Django itself and doesn't need to be validated, which I think is why the docs say to omit `include()` in that case.